lib/types/custom.js

```diff
@@ -0,0 +1,18 @@
+import url from 'url';
+import getDimensions from '../dimensions';
+
+export const parse = ([elm]) => {
+  if (elm.tagName.toLowerCase() !== 'iframe') {
+    return null;
+  }
+
+  const src = elm.getAttribute('src');
+  const {width, height} = getDimensions(elm);
+  if (!src || !width || !height) {
+    return null;
+  }
+
+  const parsed = url.parse(src, false, true);
+  const secure = parsed.protocol === 'https:';
```

Comment:
oh right, thanks. Saw now that I was exporting this from an old commit in html-to-article-json :)